src/main/resources/assets/nutrition/lang/pl_PL.lang

```diff
@@ -0,0 +1,22 @@
+# Keys
+key.nutrition=Otwórz GUI Odżywiania
+
+# GUI
+gui.nutrition:nutrition_title=Odżywianie
+gui.nutrition:close=Zamknij
+gui.nutrition:updating=Aktualizowanie...
+
+# Nutrients
+nutrient.nutrition:grain=Zboża
+nutrient.nutrition:fruit=Owoce
+nutrient.nutrition:vegetable=Warzywa
+nutrient.nutrition:protein=Białka
+nutrient.nutrition:dairy=Nabiał
+
+# Tooltips
+tooltip.nutrition:nutrients=Składniki Odżywcze:
+
+# Potions
+potion.nutrition:toughness=Twardość
```

Comment:
Nie jestem w stanie domyślić się kontekstu, proszę o więcej informacji

Reply:
Ogółem to raczej jest efekt nakładany na gracza, ale nie mogłem natrafić na ten tekst wewnątrz gry

Reply:
Myślę, że może "Wytrzymałość" albo "Odporność" może być w sumie lepsza

Reply:
![obraz](https://github.com/GTNewHorizons/Nutrition/assets/20343738/da395f5a-094e-457b-9132-b92599a0818e)
Mam, taka ikonka przy tym jest. Gdy jest się najedzonym